core/src/main/scala/scalaz/MonadTrans.scala

```diff
@@ -15,6 +15,9 @@ trait MonadTrans[F[_[_], _]] {
   /** The [[scalaz.Monad]] implied by this transformer. */
   implicit def apply[G[_] : Monad]: Monad[F[G, ?]]
 
+  def mapF[G[_], A, B](fa: F[G, A])(f: A => G[B])(implicit M: Monad[G]): F[G, B] =
+    Monad[F[G, ?]].bind(fa)(a => liftM(f(a)))
+
```

Comment:
Maybe just switching the names of type parameters `F` and `G` in `MonadTrans` would help a little? And rename the type parameter to `liftM` to `M`?